src/item/Group.js

```diff
@@ -169,10 +169,11 @@ var Group = Item.extend(/** @lends Group# */{
     },
 
     _getBounds: function _getBounds(matrix, options) {
-        var clipItem = this._getClipItem();
+        var clipItem = this._getClipItem(),
+            _matrix = clipItem && clipItem._matrix;
         return clipItem
             ? clipItem._getCachedBounds(
-                matrix && matrix.appended(clipItem._matrix) || clipItem._matrix,
+                matrix ? matrix.appended(_matrix) : _matrix,
```

Comment:
@sasensi I didn't notice that `clipItem` can be null here... In that case it would be better to either:

- Keep accessing `clipItem._matrix` like you did, or:
- rename `_matrix` to `clipMatrix`

You choose :)

Reply:
I choose original solution.
It's great, I'm improving my git skills with these code reviews ;)